benchmarks/CMakeLists.txt

```diff
@@ -1,8 +1,9 @@
 # Setup for running Google Benchmarks (https://github.com/google/benchmark) on
-# flatbuffers. This requires both that benchmark library and its depenency gtest
+# flatbuffers. This requires both that benchmark library and its dependency gtest
 # to build. Instead of including them here or doing a submodule, this uses
 # FetchContent (https://cmake.org/cmake/help/latest/module/FetchContent.html) to
 # grab the dependencies at config time. This requires CMake 3.14 or higher.
+
 cmake_minimum_required(VERSION 3.14)
```

Comment:
Should this be brought down to 3.8?

Reply:
no, see the comment above this line. It states that in this file `FetchContent` functionality is used requiring 3.14.

This file is used for benchmarks which are not required by 99% of open source users, so it's optional functionality of this project which may require higher cmake version. In this particular case, if I with my cmake 3.8 will enable the `FLATBUFFERS_BUILD_BENCHMARKS` cmake option, it will raise an error that for this particular functionality I need cmake 3.14 or higher. 

It allows 99% of open source users not to be forced by cmake requirements of optional functionality (`FLATBUFFERS_BUILD_BENCHMARKS`).

Reply:
Ok, i just don't think we need the requires cmake 3.8 comment then if its below the minimum already.